mu4/scores/internal/openscorecontroller.cpp

```diff
@@ -23,19 +23,29 @@
 
 using namespace mu;
 using namespace mu::scores;
+using namespace mu::domain::notation;
 
 void OpenScoreController::init()
 {
     dispatcher()->reg(this, "file-open", this, &OpenScoreController::openScore);
     dispatcher()->reg(this, "file-import", this, &OpenScoreController::importScore);
+    dispatcher()->reg(this, "file-newscore", this, &OpenScoreController::newScore);
```

Comment:
Is this the former name of the action? Does it match the name in `shortcuts.xml`? It needs to match

Reply:
Thank. I fixed it: file-new. Also added a call by pressing the hotkey on Ctrl+N. As it was before.